src/wp-content/themes/twentynineteen/style-editor.scss

```diff
@@ -9,6 +9,22 @@ Twenty Nineteen Editor Styles
 
 /** === Editor Frame === */
 
+.has-small-font-size {
```

Comment:
Ported the font sizes here. This was a bug on its own: the theme needs to enqueue the font size classes, which TwentyNineteen didn't do, forcing Gutenberg to add inline styles to the blocks to make it work.